Sources/UberAuth/Errors/UberAuthError.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  Copyright © Uber Technologies, Inc. All rights reserved.
+//
+
+
+import AuthenticationServices
+import Foundation
+
+public enum UberAuthError: Error {
+    // The user cancelled the auth flow
+    case cancelled
+    
+    // The application failed to open the Uber client app
+    case couldNotOpenApp(UberApp)
+    
+    // The auth code was not found or is malformed
+    case invalidAuthCode
+    
+    // Failed to build the auth request
+    case invalidRequest
+    
+    // An OAuth standard error occurred
+    case oauth(OAuthError)
```

Comment:
NIT: `oauth` -> `oAuth` for consistency with other usages.

Reply:
Fixed